src/CMakeLists.txt

```diff
@@ -142,6 +137,11 @@ ELSE ()
 ENDIF()
 
 IF (USE_HTTPS)
+	IF (CMAKE_SYSTEM_NAME MATCHES "Darwin")
+		FIND_PACKAGE(Security)
+		FIND_PACKAGE(CoreFoundation)
+	ENDIF()
+
```

Comment:
Can't this even be moved into `STREQUAL "SecureTransport"`? Or is it required somewhere else?

Reply:
I need to have checked for those so the auto-detection works (line 147 below). The other backends have their check before (OpenSSL is at line 125, line 103 is where we just trust WinHTTP is available).